receiver/sqlserverreceiver/logs-documentation.md

```diff
@@ -0,0 +1,16 @@
+# Top-Query Collection Exported Logs Attributes
+
+| Attributes               | Description                                                                                                                                                                                                                                                                                                        | Type   |   |   |   |   |   |   |   |
+|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------|---|---|---|---|---|---|---|
+| sqlserver.total_elapsed_time    | Total elapsed time, reported in microseconds (but only accurate to milliseconds), for completed executions of this plan.<br>                                                                                                                                                                                       | int    |   |   |   |   |   |   |   |
```

Comment:
IIUC, we are reporting milliseconds, not microseconds

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37958/files#diff-82ce9c3c07ed1c4e3d020692468d443c68fbbaaef382fc15a4b6b862f8205fabR368-R370

Reply:
updated